src/AceEditor/madie-ace-editor.tsx

```diff
@@ -293,7 +293,7 @@ const MadieAceEditor = ({
         <span>{editorAnnotations.length} issues were found with CQL...</span>
       );
     } else {
-      return <span>Parsing complete, CQL is valid</span>;
+      return <span data-testId="success-debounce"></span>;
```

Comment:
even though the scope of the story is to remove the success message, I think we do not need the whole footerdiv or this method. what do you think?

Reply:
What would be an alternative solution? The span is still present, but does not hold any text

Reply:
we already have an alternative. we display success/errors/warnings in the gutter and at the top of the editor, the whole footer seems to be redundant.

Reply:
I'll suggest we completely remove this gutter in favor of consolidated alerts